.github/actions/wfc_pr_management.yml

```diff
@@ -0,0 +1,67 @@
+name: WFC PR Management
```

Comment:
WFC stands for "Workflow Call" which this isn't anymore so I'd call this just PR management. Also maybe this should be better named "auto-merge-approve" maybe?

Reply:
```suggestion
name: PR management composite
```